src/pages/FarmV3/FarmTableComponents/FarmItems.tsx

```diff
@@ -0,0 +1,53 @@
+import { SSLToken } from '@/pages/FarmV3/constants'
+import React, { FC, useState } from 'react'
+import { Accordion, AccordionItem } from 'gfx-component-lib'
+import { StatsModal } from '@/pages/FarmV3/StatsModal'
+import NoResultsFound from '@/pages/FarmV3/FarmTableComponents/NoResultsFound'
+import FarmContent from '@/pages/FarmV3/FarmTableComponents/FarmTableBalanceItem'
+import { useSSLContext } from '@/context'
+import { truncateBigString } from '@/utils'
+
+const FarmItems: FC<{
+  tokens: SSLToken[]
+  numberOfCoinsDeposited: number
+  searchTokens: string
+  showDeposited: boolean
+}> = ({ tokens, numberOfCoinsDeposited, searchTokens, showDeposited }) => {
+  const [statsModal, setStatsModal] = useState<boolean>(false)
+  const { filteredLiquidityAccounts } = useSSLContext()
+  return (
+    <div className={''}>
+      {numberOfCoinsDeposited === 0 && showDeposited && searchTokens?.length === 0 && (
```

Comment:
on line 20 you do exactly searchTokens non-nullable check.

Reply:
good point, this can just be compounded as A & (!B || C&B)  which inturn i reckon is just A & B